src/components/common/Extensions/UserHub/partials/TransactionsTab/partials/TransactionStatus.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import { format } from 'date-fns';
+import { TRANSACTION_STATUSES } from '~types';
```

Comment:
Add an empty line between imports packages and other imports (like hooks, types, components, etc)